fpm/src/fpm_model.f90

```diff
@@ -111,4 +112,47 @@ module fpm_model
         ! Native libraries to link against
 end type fpm_model_t
 
+contains
+
+function pickle_srcfile(source) result(s)
```

Comment:
Is there a specific reason to call it `pickle`? This is probably inspired by the Python `pickle` functionality, but do we really mirror this?

Reply:
I know, I thought about it too. Can you suggest a better name?

Reply:
The question is what is it supposed to mean? From its functionality I would suggest `show_srcfile`.